src/_assets/css/_site_overrides.scss

```diff
@@ -0,0 +1,3 @@
+// Flutter-site specific style overrides
```

Comment:
@fertolg - note this include is to be used for site-specific overrides (the plan is to share the rest of the Sass include files across sites, and place all site-specific overrides in this file -- or other includes, but imported here). /cc @filiph